clients/src/main/java/org/apache/kafka/common/requests/FindCoordinatorResponse.java

```diff
@@ -107,6 +107,14 @@ public int throttleTimeMs() {
         return throttleTimeMs;
     }
 
+    public boolean hasError() {
+        return this.error != Errors.NONE;
+    }
+
+    public boolean retriableError() {
+        return error == Errors.COORDINATOR_NOT_AVAILABLE || error == Errors.COORDINATOR_LOAD_IN_PROGRESS;
```

Comment:
Can we update the possible error codes at the top of this file? It does not include `COORDINATOR_LOAD_IN_PROGRESS` and it incorrectly lists `NOT_COORDINATOR`.